src/javascripts/components/color.js

```diff
@@ -85,6 +85,18 @@ export default class extends React.Component {
     )
   }
 
+  _savedNotification(){
+    let layout = this.props.layout
+    let label = this.props.label
+    let saved = this.props.saved
+    let savedInline = span({
+      className: "frigb-saved-inline frigb-colorpicker-inline"},
+      "saved")
```

Comment:
Positioning of the brackets is hard makes the nesting non-obvious. Also it's inconsistent with the rest of the project. Please change to:
```
let className = "frigb-saved-inline frigb-colorpicker-inline"
let savedInline = span({className}, "saved")
```